lib/pure/asynchttpserver.nim

```diff
@@ -404,6 +404,7 @@ proc acceptRequest*(server: AsyncHttpServer,
 proc serve*(server: AsyncHttpServer, port: Port,
             callback: proc (request: Request): Future[void] {.closure, gcsafe.},
             address = "";
+            domain: Domain = AF_INET;
```

Comment:
Move this parameter after `assumedDescriptorsPerRequest` please.

Reply:
Done